CMakeLists.txt

```diff
@@ -266,4 +266,6 @@ if (BUILD_DOCS)
     # and if it doesn't, clearly state why it fails
     message( FATAL_ERROR "Expected to find xelatex at: ${XELATEX} ; check your LaTeX installation")
   endif()
+  set(TEX_INTERACTION "batchmode" CACHE STRING "Choose the interaction mode for TeX." FORCE)
+  set_property(CACHE TEX_INTERACTION PROPERTY STRINGS "nonstopmode" "batchmode")
```

Comment:
Developers will now default to batchmode for a cleaner stdout during builds, but they can switch to nonstopmode easily.